integration/securitypolicy_test.go

```diff
@@ -1082,3 +1082,142 @@ func TestSecurityPolicyWithContextRef(t *testing.T) {
 
 	testenv.Test(t, policyAndOperatorCtx)
 }
+
+func TestUpdateStatusOfLinkedAPIs(t *testing.T) {
+	var (
+		api1    = "api1"
+		api2    = "api2"
+		testNs  = ""
+		polName = ""
+		pol     *v1alpha1.SecurityPolicy
+	)
+
+	f := features.New("UpdateStatusOfLinkedAPIs").
+		Setup(func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+			eval := is.New(t)
```

Comment:
what do you think having a function scope `eval` instead of creating a new instance on each features?

Reply:
Will do